### PR TITLE
Upgraydde storybook #minor

### PR DESCRIPTION
**Pull request by GiantRobots**
## Merge Checklist

- [ ] Includes link to associated Github Issue or JIRA ticket
- [ ] Has descriptive PR title and body
- [ ] All changes are backwards compatible
- [ ] QA'd by the SendGrid Style Guide designer
- [ ] Code reviewed by a SendGrid frontend lead
- [ ] If adding a new component, the component has been added to the exports in `src/index.ts`.

### CODE REVIEW

**Comment by GiantRobots** [comment]
`npm install` and fix broken dependencies. All of the tests work and the image-snapshots that were updated were mostly around font/glyph/shadow rendering and then I skipped a test that is constantly failing. The flex header thing had an issue with the screenshot that had different widths on the regular, I skipped it because I'm tired of it breaking but it could probably be resolved by updating the storybook test.

File: .buildkite/buildPipeline.yml [hunk 1]
```diff
@@ -9,62 +8,62 @@ steps:
...
         push:
-        - build:docker.sendgrid.net/sendgrid/ui-components:latest
+          - build:docker.sendgrid.net/sendgrid/ui-components:latest
 
   - wait
-  
-  - name: ":jest: Lint and Test"
-    command: 
+
+  - name: ':jest: Lint and Test'
```
Comment (dan-sg) [inline .buildkite/buildPipeline.yml]: 😄 `single quotes`

File: src/text-input.tsx [hunk 2]
```diff
@@ -247,9 +247,10 @@ const initState = (props: TextInputProps) => {
 
 export type BaseProps = TextInputProps & HTMLInputElementProps;
 
+// I think this component should be burned in a fire 🔥🔥🔥 - Mr. t
```
Comment (dan-sg) [inline src/text-input.tsx]: remove this one.

**Comment by thinkingserious** [comment]
Hello @GiantRobots,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**